src/client/components/dimension-measure-panel/dimension-measure-panel.tsx

```diff
@@ -117,11 +117,12 @@ export class DimensionMeasurePanel extends React.Component<DimensionMeasurePanel
           style={dimensionListStyle}
         />
         {showResizeHandle && <ResizeHandle
+          renderIcon={true}
```

Comment:
Idea for consideration: maybe add prop `as` and in this case pass `<SvgIcon>`?

Reply:
Oh, yeah you are right. Makes much more sense just to render the icon as a child 😸

Reply:
`children` is even better!